### PR TITLE
Added a job to renew user subscriptions

### PR DESCRIPTION
**Pull request by levb**
#### Summary
- Added a simple job to renew all user subscriptions every 24 hours
- Increased the default subscription expiry to 48 hours

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-mscalendar/issues/54

### CODE REVIEW

File: server/jobs/renew_job.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+func NewRenewJob() RegisteredJob {
+	return RegisteredJob{
+		id:       "renew",
+		interval: 24 * time.Hour,
+		work:     runRenewJob,
+	}
+}
+
+// runDailySummaryJob delivers the daily calendar summary to all users who have their settings configured to receive it now
```
Comment (mickmister) [inline server/jobs/renew_job.go:22]:
```suggestion
// runRenewJob calls renews the event subscription for each connected user
```

File: server/jobs/job_manager.go [hunk 2]
```diff
@@ -70,7 +70,7 @@ func (jm *JobManager) OnConfigurationChange(env mscalendar.Env) error {
 
 	jm.registeredJobs.Range(func(k interface{}, v interface{}) bool {
 		job := v.(RegisteredJob)
-		enabled := job.isEnabledByConfig(env)
+		enabled := (job.isEnabledByConfig == nil) || job.isEnabledByConfig(env)
```
Comment (larkox) [inline server/jobs/job_manager.go:73]: Why not this `or` logic inside the "isEnableByConfig" logic?

Comment (levb) [reply server/jobs/job_manager.go:73]: 'cause providing a function for the "always on" case seemed wrong. nil is more appropriate?

**Pull request merged**